addons/udes_stock/models/stock_picking.py

```diff
@@ -177,7 +177,7 @@ def get_empty_locations(self, limit=None, sort=True):
         :returns: Move lines of picking
         """
         locations = self._get_child_dest_locations(
-            aux_domain=[("barcode", "!=", False), ("quant_ids", "=", False)], limit=limit
+            [("u_blocked", "=", False), ("barcode", "!=", False), ("quant_ids", "=", False)], limit=limit
```

Comment:
Actually, can we add a function to get the domain to use in this search. Then we can extend it in udes_simple_blocking to add the `u_blocked` parameter and remove the need for udes_stock to depend upon udes_simple_blocking. It fits more nicely to have udes_simple_blocking depend upon udes_stock instead.

Reply:
Wouldn't that create a circular dependency?

Reply:
No because udes_stock wouldn't depend upon blocked locations. Blocked locations would just override the domain.

Reply:
Ah, sorry, I see what you mean. At the moment it's udes_stock that depends on simple_location_blocking but you want to move the functionality hence the dependency. Makes sense. Will do.